CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+- Replaces [_golint_](https://github.com/golang/lint) with [_revive_](https://github.com/mgechev/revive)
```

Comment:
Personally, I think it is not worth putting it in the changelog. It is purely internal.